### DISCUSSION TITLE
How to accessing local file paths directly in Uno Platform and WebAssembly

### DISCUSSION THREAD

**Discussion by shouresh110** [unoplatform/uno#17780] [Q&A] [answered]
When working with Uno Platform and WebAssembly, accessing local file paths directly (like "D:\MyFolder\Image1.png") isn't feasible because WebAssembly runs in a browser environment, which restricts direct access to the user's filesystem for security reasons.I want Find a way to solve it.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. 

You can find information on [the webassembly file system](https://platform.uno/docs/articles/features/file-management.html#webassembly-file-system) and [FileOpenPicker](https://platform.uno/docs/articles/features/windows-storage-pickers.html).
